moveit_planners/ompl/ompl_interface/src/detail/threadsafe_state_storage.cpp

```diff
@@ -61,6 +61,7 @@ moveit::core::RobotState* ompl_interface::TSStateStorage::getStateStorage() cons
   if (it == thread_states_.end())
   {
     st = new moveit::core::RobotState(start_state_);
+    st->setToDefaultValues();  // avoid uninitialized memory
```

Comment:
Please see https://github.com/ros-planning/moveit/pull/2273#discussion_r631760679 .
In addition to removing this line, it's probably worth to port the new ROS2 test for it?

Reply:
Which would be the cleanest git procedure to port the test? Cherry picking?

Reply:
done